go/pkg/repository/cached.go

```diff
@@ -37,9 +37,9 @@ func NewCachedRepository(repo Repository, cachePrefix string, projectDir string,
 }
 
 // NewCachedMetadataRepository returns a CachedRepository that caches the metadata/ path in
-// .replicate/metadata-cache in a source dir
+// .keepsake/metadata-cache in a source dir
 func NewCachedMetadataRepository(projectDir string, repo Repository) (*CachedRepository, error) {
-	return NewCachedRepository(repo, "metadata", projectDir, ".replicate/metadata-cache")
+	return NewCachedRepository(repo, "metadata", projectDir, ".keepsake/metadata-cache")
```

Comment:
Do we care about blowing away the cache? I guess metadata cache is small so it doesn't matter if we leave `.replicate/metadata-cache` kicking around.